packages/backend/src/models/SpaceModel.ts

```diff
@@ -633,6 +633,59 @@ export class SpaceModel {
         });
     }
 
+    async getSpacesForAccessCheck(
```

Comment:
this can be reused across many other controllers/queries, but keeping it just for /availableFilters on this PR for now.